cargo.eclass set rustflags as string in ${CARGO_HOME}/config.toml
zed-industries/zed/.cargo/config.toml set rustflags as array
Cargo: failed to merge config value from `${WORKDIR}/cargo_home/config.toml` into
`${WORKDIR}/zed-0.147.2/.cargo/config.toml`: expected array, but found string

diff --git a/.cargo/config.toml b/.cargo/config.toml
index a657ae61b9..0236ebaf53 100644
--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -1,18 +1,18 @@
 [build]
 # v0 mangling scheme provides more detailed backtraces around closures
-rustflags = ["-C", "symbol-mangling-version=v0", "--cfg", "tokio_unstable"]
+rustflags = "-C symbol-mangling-version=v0 --cfg tokio_unstable"
 
 [alias]
 xtask = "run --package xtask --"
 
 [target.x86_64-unknown-linux-gnu]
 linker = "clang"
-rustflags = ["-C", "link-arg=-fuse-ld=mold"]
+rustflags = "-C link-arg=-fuse-ld=mold"
 
 [target.aarch64-unknown-linux-gnu]
 linker = "clang"
-rustflags = ["-C", "link-arg=-fuse-ld=mold"]
+rustflags = "-C link-arg=-fuse-ld=mold"
 
 # This cfg will reduce the size of `windows::core::Error` from 16 bytes to 4 bytes
 [target.'cfg(target_os = "windows")']
-rustflags = ["--cfg", "windows_slim_errors"]
+rustflags = "--cfg windows_slim_errors"
-- 
2.45.2

